packages/dev/core/src/ShadersWGSL/ShadersInclude/lightFragment.fx

```diff
@@ -5,9 +5,9 @@
         #ifdef PBR
             // Compute Pre Lighting infos
             #ifdef SPOTLIGHT{X}
-                preInfo = computePointAndSpotPreLightingInfo(light{X}.vLightData, viewDirectionW, normalW, vPositionW);
+                preInfo = computePointAndSpotPreLightingInfo(light{X}.vLightData, viewDirectionW, normalW, input.vPositionW);

```

Comment:
Is it not `fragmentInputs` instead of `input` (several occurrences)?

Reply:
yeah I know, I used them both. Both are defined